### PR TITLE
Add links to all issues in the 3.6.0 section of the changelog

### PR DESCRIPTION
**Pull request by rodrigoprimo**
## Description

This PR adds links to all issues in the 3.6.0 section of the changelog. Three typos were also fixed in a separate commit.

## Related issues/external references

Part of #244

## Types of changes

- [ ] Bug fix _(non-breaking change which fixes an issue)_
- [ ] New feature _(non-breaking change which adds functionality)_
- [ ] Breaking change _(fix or feature that would cause existing functionality to change)_
    - [ ] This change is only breaking for integrators, not for external standards or end-users.
- [x] Documentation improvement

## PR checklist

- [x] I have checked there is no other PR open for the same change.
- [x] I have read the [Contribution Guidelines](https://github.com/PHPCSStandards/PHP_CodeSniffer/blob/master/.github/CONTRIBUTING.md).
- [x] I grant the project the right to include and distribute the code under the BSD-3-Clause license (and I have the right to grant these rights).
- [ ] I have added tests to cover my changes.
- [ ] I have verified that the code complies with the projects coding standards.
- [ ] [Required for new sniffs] I have added XML documentation for the sniff.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -610,7 +610,7 @@ Additionally, thanks to [Alexander Turek][@derrabus] for consulting on the repo
 - Include patterns are now ignored when processing STDIN
     - Previously, checks using include patterns were excluded when processing STDIN when no file path was provided via --stdin-path
     - Now, all include and exclude rules are ignored when no file path is provided, allowing all checks to run
-    - If you want include and exclude rules enforced when checking STDIN, use --stdin-path to set the file path
+    - If you want to include and exclude rules enforced when checking STDIN, use --stdin-path to set the file path
```
Comment (jrfnl) [inline CHANGELOG.md:613]: This `to` changes the meaning of the sentence and is incorrect.

**Review by jrfnl — CHANGES_REQUESTED** [review]
Thanks @rodrigoprimo ! One thing to change before this can be merged.

As these are quite simple PRs, feel free to just use one commit for these type of changes.

File: CHANGELOG.md (hunk 1 above)
Comment (rodrigoprimo) [reply CHANGELOG.md]: I just fixed this by amending the original commit.

**Review by jrfnl — APPROVED** [review]
Thanks @rodrigoprimo !

**Pull request merged**